school/redfish/computersystem.go

```diff
@@ -363,7 +363,7 @@ type ComputerSystem struct {
 	AssetTag string
 	// bios shall be a link to a resource of
 	// type Bios that lists the Bios settings for this system.
-	bios string
+	bios common.Link
```

Comment:
It would be preferable to keep these strings and follow the pattern used in other object definitions to have the extraction handled in the JSON decoding.